### ISSUE TITLE
[Feature Request] Macromedia Dreamweaver MX 2004 Icon

### ISSUE THREAD

**Issue by joshmanders** [Idered/chalk.ist#62]
I usually put some silly filename in the title of my images as a little easter egg and right now the icon I have chosen is the green VSCode one, but one of my silly shticks is that I say I still use Macromedia Dreamweaver MX 2004 as my editor of choice and I would really love to carry that over into these screenshots, it would probably cause more eyeballs tbh lol.

**Comment by Idered** [comment]
Hey Josh, I can't find that icon in svg format. I can add support for custom png/svg icons but right now I'm focused on adding code animations and [video exports](https://bsky.app/profile/idered.bsky.social/post/3lcklos6jbc23) so that will have to wait. 

If you have time to PR this, that would be awesome. I think that all we need is file input in icon picker dropdown, once icon is selected, just base64 it and save in directly in code block object - this would allow to have custom icon per code block

**Comment by joshmanders** [comment]
@Idered yeah no problem when I get some downtime I'll play with it and get a PR open. Thanks!
